packages/swapper/src/api.ts

```diff
@@ -188,6 +190,8 @@ export enum SwapErrorTypes {
   TRADE_FAILED = 'TRADE_FAILED'
 }
 
+export type SwapSupportedUtxoChainIds = ThorChainUtxoChainIds
```

Comment:
Can we follow same naming convention as evm and cosmossdk as `UtxoSupportedChainIds`. And in retrospect, I think these unioned chain id types should be singular vs plural